src/components/common/TmBalance.vue

```diff
@@ -19,7 +19,10 @@
                 {{ overview.totalStake | bigFigureOrShortDecimals | noBlanks }}
               </h2>
             </div>
-            <div v-if="isMultiDenomNetwork" class="currency-selector">
+            <div
+              v-if="isMultiDenomNetwork && stakingBalance.fiatValue"
```

Comment:
HACK

Reply:
it actually makes a lot of sense, I think.

But I guess you could call every piece of software ever written in human History a "hack"

Reply:
äh no ;P

Reply:
xD